devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusPlatformTask.java

```diff
@@ -0,0 +1,60 @@
+package io.quarkus.gradle.tasks;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Properties;
+
+import io.quarkus.platform.descriptor.QuarkusPlatformDescriptor;
+import io.quarkus.platform.descriptor.resolver.json.QuarkusJsonPlatformDescriptorResolver;
+import io.quarkus.platform.tools.config.QuarkusPlatformConfig;
+
+public class QuarkusPlatformTask extends QuarkusTask {
+
+    QuarkusPlatformTask(String description) {
+        super(description);
+    }
+
+    protected void setupPlatformDescriptor() {
+
+        if (QuarkusPlatformConfig.hasGlobalDefault()) {
+            return;
+        }
+
+        final Path currentDir = getProject().getProjectDir().toPath();
+
+        final Path gradlePropsPath = currentDir.resolve("gradle.properties");
+        if (Files.exists(gradlePropsPath)) {
+            final Properties props = new Properties();
+            try (InputStream is = Files.newInputStream(gradlePropsPath)) {
+                props.load(is);
+            } catch (IOException e) {
+                throw new IllegalStateException("Failed to load " + gradlePropsPath, e);
+            }
+
+            final QuarkusPlatformDescriptor platform = QuarkusJsonPlatformDescriptorResolver.newInstance()
+                    .setArtifactResolver(extension().resolveAppModel())
+                    .setBomVersion(
+                            getRequiredProperty(props, "quarkusPlatformBomGroupId"),
```

Comment:
these shuld be something like `quarkus.platform.bom.group-id` to match naming style use elsewhere.

i.e. quarkus.version as used in mvn.

maybe just quarkus.platform.group-id to make it shorter; the bom is implied anyways.

Reply:
No objections changing the style. We've been using quarkusVersion in gradle.properties, so I simply followed that.
As to `quarkus.platform.group-id` w/o the `bom` in it, possibly ok, however we might want to support `quarkus.platform.json.group-id` at some point. QuarkusJsonPlatformDescriptorResolver for example already includes such a property.

Reply:
I'm gonna merge it in this state though. Adjusting the style could be done as a separate PR.

Reply:
I think camel case is the usual Gradle style. At least I saw that used a lot in Gradle files.